src/RNCamera.js

```diff
@@ -256,6 +262,14 @@ export default class Camera extends React.Component<PropsType, StateType> {
     };
   }
 
+  async setBrightnessAsync(brightness) {
```

Comment:
doe we really need these methods if the brightness can already be controlled with the brightness property?